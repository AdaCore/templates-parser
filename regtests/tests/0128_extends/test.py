from test_support import *
run('testme', ["extends.thtml"])
