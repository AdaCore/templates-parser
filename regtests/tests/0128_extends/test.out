Param is Param1
<div class="block1">
The block1 Param2
</div>
<div class="block2">
The block2 Param2
</div>
<div class="block3">
default for Param1 block3
</div>
Some text in including file
